src/main/java/io/ebean/common/ModifyHolder.java

```diff
@@ -78,11 +90,25 @@ void modifyRemoval(Object bean) {
   }
 
   Set<E> getModifyAdditions() {
```

Comment:
Better: return a list here, as a list is more efficient
```suggestion
  List<E> getModifyAdditions() {
```

Reply:
This would mean changing the interface of BeanCollection from Set to List twice, resulting in ~100 changes over all Ebean. Do you really want this? @rPraml

Reply:
you are right. We should not change that part of the API now. Maybe we can change this in a later PR when it is part of the upstream